src/_bentoml_impl/loader.py

```diff
@@ -123,16 +124,19 @@ def import_service(
         extra_python_path = None
 
     # patch model store if needed
-    if (
-        bento_path.parent.joinpath(BENTO_YAML_FILENAME).exists()
-        and bento_path.parent.joinpath("models").exists()
-    ):
+    if (bento_parent_dir := bento_path.parent).joinpath(
+        BENTO_YAML_FILENAME
+    ).exists() and bento_parent_dir.joinpath("models").exists():
         from bentoml._internal.models import ModelStore
 
         original_model_store = BentoMLContainer.model_store.get()
+        # cwd into this for relative path to work
+        os.chdir(bento_parent_dir.absolute())
```

Comment:
I think we should restore the cwd after import

Reply:
updated